packages/phrases/src/locales/en/translation/admin-console/application-details.ts

```diff
@@ -28,7 +29,7 @@ const application_details = {
   add_another: 'Add Another',
   id_token_expiration: 'ID Token expiration',
   refresh_token_expiration: 'Refresh Token expiration',
-  token_endpoint: 'Token endpoint',
+  token_endpoint: 'Token Endpoint',
```

Comment:
Why do we need to change the letter case here? If we do, we should also change it for others, like the one below "Userinfo endpoint", right?

Reply:
because it's a term - and yes we should change all of them

Reply:
created LOG-4231